client/gui/CAnimation.h

```diff
@@ -44,6 +45,8 @@ class IImage
 
 	//only indexed bitmaps, 16 colors maximum
 	virtual void shiftPalette(int from, int howMany) = 0;
+	virtual void adjustPalette(ColorShifter * shifter) = 0;

```

Comment:
`ColorShifter * shifter` should be `const`

Reply:
Updated.